nucliadb_node/src/bin/writer.rs

```diff
@@ -108,47 +108,55 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         }
     });
 
-    if let Some(prometheus_url) = Configuration::get_prometheus_url() {
+    let report = NodeReport::new(host_key.to_string())?;
+
+    let metrics_task = tokio::spawn(async move {
         info!("Start metrics task");
 
-        let report = NodeReport::new(host_key.to_string())?;
-        let mut metrics_publisher = Publisher::new("node_metrics", prometheus_url);
+        let mut node_reader = NodeReaderService::new();
 
-        if let Some((username, password)) =
-            Configuration::get_prometheus_username().zip(Configuration::get_prometheus_password())
-        {
-            metrics_publisher = metrics_publisher.with_credentials(username, password);
-        }
+        if let Err(e) = node_reader.load_shards() {
```

Comment:
load shards will open all the shards, is there a way to avoid this?

Reply:
Alternative implemented at [2164223](https://github.com/nuclia/nucliadb/pull/486/commits/2164223a5d2ff1aa68d4920b4ebe69424aaecbf2)